src/main.c

```diff
@@ -1074,6 +1178,86 @@ static gchar* r_status_formatter_json(gboolean pretty)
 #endif
 }
 
+static RaucSlotStatus* r_variant_get_slot_state(GVariant *vardict)
+{
+	RaucSlotStatus *slot_state = g_new0(RaucSlotStatus, 1);
+	GVariantDict dict;
+
+	g_variant_dict_init(&dict, vardict);
+
+	g_variant_dict_lookup (&dict, "bundle.compatible", "s", &slot_state->bundle_compatible);
```

Comment:
Remove the space before (, and in the following lines as well.

Reply:
Oh, I missed that originally. Fixed it now!